### DISCUSSION TITLE
Bug: Final sprite texture is wrong if the parent has the coordinates changed

### DISCUSSION THREAD

**Discussion by viniciusbitt** [pixijs/pixijs#9192] [Q&A] [answered]
### Current Behavior

When I click on the screen the bunny is added to the coordinates where I clicked, but if the `parent` has its coordinates changed, the bunny will be in another coordinates, different from where I clicked.

### Expected Behavior

I believe that the bunny should be in the same coordinates where I clicked, since I'm using the `app.stage.extract...` only on the `landscape` texture.

### Steps to Reproduce

Click anywhere on the landscape, comment out the `sprite` coordinates changes to see it working, and uncomment to see the bug.

[Pixi-Playground.webm](https://user-images.githubusercontent.com/34211799/220004470-3f6da643-066d-4db2-8226-2053a61e51e4.webm)

```javascript
const landscape_texture = await PIXI.Assets.load(
    "https://cdn.vox-cdn.com/thumbor/nLYEvRNQpPRDUDepKscMcDkqwB8=/0x0:3402x2551/1200x800/filters:focal(1429x1004:1973x1548)/cdn.vox-cdn.com/uploads/chorus_image/image/66468459/Landscaping-iStock-498015683.0.0.jpg"
);
const bunny_texture = await PIXI.Assets.load("https://pixijs.io/examples/examples/assets/bunny.png");

const sprite = new PIXI.Sprite();
app.stage.addChild(sprite);
let landscape = new PIXI.Sprite(landscape_texture);
sprite.addChild(landscape);

//the problem is here, comment and it will work
sprite.x = -150; sprite.y = -150;

sprite.interactive = true;

sprite.on("pointerdown", async (e) => {
    const { x, y } = e.getLocalPosition(sprite);

    const bunny = new PIXI.Sprite(bunny_texture);
    bunny.x = x - bunny.width / 2;
    bunny.y = y - bunny.height / 2;
    landscape.addChild(bunny);

    const landscape_updated = new PIXI.Sprite(PIXI.Texture.from(await app.renderer.extract.image(landscape)))
    sprite.removeChild(landscape);
    landscape = landscape_updated;
    sprite.addChild(landscape);
});
```

### Environment

- **`pixi.js` version**: *7.2.0-beta*
- **Browser & Version**: *Edge Version 110.0.1587.50 (Official build) (64-bit)*
- **OS & Version**: *Windows 11*
- **Running Example**: *<https://pixiplayground.com/#/edit/H0W4SoNp9psLZXz46YyZS>*

**Comment by bigtimebuddy** [comment, marked as answer]
https://pixiplayground.com/#/edit/al0bUCB15uESga2PQiH4w

The quick fix is to do `sprite.removeChild(landscape)` _before_ calling extract. Extract plugin considers the world transform which is being applied to the rendering of the children, if you detach `landscape`, it doesn't apply the world transform to the bunny child.

That said, this is not the best way to do achieve this. You should be using a RenderTexture if you're going to stamp images like this on another image. It's much easier to account for the transform than using extract and it's much more memory friendly.
